CefSharp/Internals/MethodParameter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
```

Comment:
- Add license disclaimer
- Remove redundant using statements
- Make sure formatting is `spaces` (not `tabs` as `VS` typically uses as the default).